crates/net/network-api/src/noop.rs

```diff
@@ -35,6 +35,7 @@ impl NetworkInfo for NoopNetwork {
                 head: Default::default(),
                 network: 1,
                 genesis: Default::default(),
+                config: Default::default(),
```

Comment:
nit: while we're at it, let's move `head` to last position, so we're matching the serialized object

Reply:
Done